### PR TITLE
Updates for 4.8 release - updateeachrelease topics Part 1

### PR DESCRIPTION
**Pull request by mairaw**
Shouldn't be merged before 4.8 is released

Contributes to #11630

### CODE REVIEW

**Comment by mairaw** [comment]
@rpetrusha this one is ready for review. The WIP in the title is to block this from merging.

File: docs/framework/get-started/index.md [hunk 1]
```diff
@@ -34,7 +33,7 @@ The .NET Framework is a managed execution environment for Windows that provides
 
 - Version compatibility. With rare exceptions, apps that are developed by using a particular version of the .NET Framework run without modification on a later version.
 
-- Side-by-side execution. The .NET Framework helps resolve version conflicts by allowing multiple versions of the common language runtime to exist on the same computer. This means that multiple versions of apps can coexist and that an app can run on the version of the .NET Framework with which it was built. Side-by-side execution applies to the .NET Framework version groups 1.0/1.1, 2.0/3.0/3.5, and 4/4.5.x/4.6.x/4.7.x.
+- Side-by-side execution. The .NET Framework helps resolve version conflicts by allowing multiple versions of the common language runtime to exist on the same computer. This means that multiple versions of apps can coexist and that an app can run on the version of the .NET Framework with which it was built. Side-by-side execution applies to the .NET Framework version groups 1.0/1.1, 2.0/3.0/3.5, and 4/4.5.x/4.6.x/4.7.x/4.8.x.
```
Comment (rpetrusha) [inline docs/framework/get-started/index.md]:
```suggestion
- Side-by-side execution. The .NET Framework helps resolve version conflicts by allowing multiple versions of the common language runtime to exist on the same computer. This means that multiple versions of apps can coexist and that an app can run on the version of the .NET Framework with which it was built. Side-by-side execution applies to the .NET Framework version groups 1.0/1.1, 2.0/3.0/3.5, and 4/4.5.x/4.6.x/4.7.x/4.8.
```

File: docs/framework/get-started/index.md [hunk 2]
```diff
@@ -49,14 +48,13 @@ In general, you shouldn't uninstall versions of the .NET Framework that are inst
...
 
 On Windows versions before Windows 8, if you do choose to remove the .NET Framework, always use **Programs and Features** from Control Panel to uninstall it. Never remove a version of the .NET Framework manually. On Windows 8 and above, the .NET Framework is an operating system component and cannot be independently uninstalled.
 
 Note that multiple versions of the .NET Framework can coexist on a single computer at the same time. This means that you don't have to uninstall previous versions in order to install a later version.
 
-<a name="ForDevelopers"></a> 
-## The .NET Framework for developers
+## <a name="ForDevelopers"></a> The .NET Framework for developers
 
 If you're a developer, choose any programming language that supports the .NET Framework to create your apps. Because the .NET Framework provides language independence and interoperability, you interact with other .NET Framework apps and components regardless of the language with which they were developed.
```
Comment (rpetrusha) [inline docs/framework/get-started/index.md]: line 63 needs modification:  The most recent production version is the .NET Framework 4.7.2. -->  The most recent production version is the .NET Framework 4.8.

File: docs/framework/migration-guide/version-compatibility.md [hunk 3]
```diff
@@ -1,68 +1,72 @@
...
-  
+- If you have a .NET Framework 1.1 app, also see [Changes in .NET Framework 2.0](https://go.microsoft.com/fwlink/?LinkID=125263).
+
+- If you're recompiling existing source code to run on the .NET Framework 4.5 or its point releases, or if you're developing a new version of an app or component that targets the .NET Framework 4.5 or its point releases from an existing source code base, check [What's Obsolete in the Class Library](../whats-new/whats-obsolete.md) for obsolete types and members, and apply the workaround described. (Previously compiled code will continue to run against types and members that have been marked as obsolete.)
+
+- If you determine that a change in the .NET Framework 4.5 has broken your app, check the [Runtime Settings Schema](../configure-apps/file-schema/runtime/index.md) to determine whether you can use a runtime setting in your app's configuration file to restore the previous behavior.
```
Comment (rpetrusha) [inline docs/framework/migration-guide/version-compatibility.md]:
```suggestion
- If you determine that a change in the .NET Framework 4.5 has broken your app, check the [Runtime Settings Schema](../configure-apps/file-schema/runtime/index.md), and particularly the [\<AppContextSwitchOverrides> Element](../configure-apps/file-schema/runtime/appcontextswitchoverrides-element.md), to determine whether you can use a runtime setting in your app's configuration file to restore the previous behavior.
```

**Review by rpetrusha — APPROVED** [review]
This looks good, @mairaw. I've left a few suggestions.

File: docs/framework/get-started/index.md (hunk 2 above)
Comment (mairaw) [reply docs/framework/get-started/index.md]: Good catch! Also changed the Windows version.

**Pull request merged**